### DISCUSSION TITLE
Is it possible to show dependencies in nvim-tree

### DISCUSSION THREAD

**Discussion by d0zingcat** [nvim-tree/nvim-tree.lua#1455] [Q&A] [answered]
As you know, when developing we need to dive into the libraries and read source codes, but when safaring(in business code, go to definition to jump into the lib), there would be a lot of embeded calls, which distributes in different files, that could be very difficult to find the code we need immediately without a tree view of the library codes.

**Comment by alex-courtis** [comment, marked as answer]
That is well outside the scope of nvim-tree, which does not and will likely never attempt to understand your source.

You can achieve your goal:

1. Add [LSP](https://neovim.io/doc/lsp/) or [coc](https://github.com/neoclide/coc.nvim) or even just tags for your project
2. setup
```lua
update_focused_file = {
      enable = true,
      update_root = true,
}
```
3. jump to definition

**Reply by d0zingcat** [reply]
That's exactly what I want, thanks a lot!
